scene/2d/physics_body_2d.cpp

```diff
@@ -55,10 +55,7 @@ PhysicsBody2D::~PhysicsBody2D() {
 }
 
 Ref<KinematicCollision2D> PhysicsBody2D::_move(const Vector2 &p_linear_velocity, bool p_test_only, real_t p_margin) {
```

Comment:
Isn't the C++ argument name wrong? It's a distance and not a velocity here if I understand correctly, so it should be `p_distance` (same in 3D).

Reply:
Yes, strangely the change was not made in the PR that I reverted (for the record the old name used was `motion`)